docs/userGuide/contentAuthoring.md

```diff
@@ -280,6 +280,55 @@ Front matter can also be included from a separate file, just like how content ca
 
 This will result in `index.md` having the title `Binary Search Tree` and the specified keywords in order for it to be looked up through the search bar.
 
-<include src="../common/userGuideSections.md" />
+### Using Footers
+
+It is optional to put footers in your site. If you wish to do so, there are two ways that MarkBind supports.
+
+#### 1. Specifying footer file in Front Matter
+
+By default, there is a `footer.md` file created for you in `_markbind/footers`. 
+If you need additional footers for different pages, you can create footer files in the same folder and wrap the contents in a `<footer>` tag. 
+
+```html
+<!-- In _markbind/footers/myFooterFile.md -->
+<footer>
+    This is my own footer!
+</footer>
+```
+
+After which, you must specify the file within [front matter](#front-matter)'s `footer` attribute to render the footer.
+
+```html
+<!-- In the file you want to include a footer -->
+<frontmatter>
+footer: myFooterFile.md

```

Comment:
Should we add two spaces indentation to make it consistent with the Front Matter section?

Reply:
Yes.